v5analytics/visallo
# Fix descriptions, some styles in find path (3.1)

Show "1 relationship" instead of "1 relationships" for example. tighten up some CSS

Testing Instructions:
1. Find path in graph
2. Select 1 edge type "button should say "1 relationship type" not "relationships"
3. First screen should show "1 relationship" not "1 relationships"
